e2e/workspace/crate1/src/lib.rs

```diff
@@ -2,5 +2,5 @@ use ts_rs::TS;
 
 #[derive(TS)]
 pub struct Crate1 {
-    pub x: i32
+    pub x: [[[i32; 128]; 128]; 128],
```

Comment:
I like how the fact that this generates stupidly large TS types endedd up being useful lol